app/views/pages/home.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="navbar-bg navbar-content d-flex space-between">
    <%= link_to "Hairbnb", "/ruby-book", class: "logo" %>
-   <%= link_to "Sign in", "/ruby-book", class: "btn" %>
+   <%= link_to "Sign in", " /users/edit", class: "btn" %>
```

Comment:
<% if user  =  non sign in %>
alors on a le link to de sign in %>
else 
on a le bouton avatar